examples/imgui_impl_dx12.cpp

```diff
@@ -46,16 +46,89 @@ static ID3D12Resource*              g_pFontTextureResource = NULL;
 static D3D12_CPU_DESCRIPTOR_HANDLE  g_hFontSrvCpuDescHandle = {};
 static D3D12_GPU_DESCRIPTOR_HANDLE  g_hFontSrvGpuDescHandle = {};
 
+static ID3D12DescriptorHeap*        g_pd3dSrvDescHeap = NULL;
+
 struct FrameResources
 {
     ID3D12Resource*     IndexBuffer;
     ID3D12Resource*     VertexBuffer;
     int                 IndexBufferSize;
     int                 VertexBufferSize;
 };
-static FrameResources*  g_pFrameResources = NULL;
+
 static UINT             g_numFramesInFlight = 0;
-static UINT             g_frameIndex = UINT_MAX;
+
+struct FrameContext
+{
+    ID3D12CommandAllocator*     CommandAllocator;
+    UINT64                      FenceValue;
+    ID3D12Resource*             RenderTarget;
```

Comment:
FenceValue is never used, would that be a problem?

Reply:
Oh, It seems it was replaced by FenseSignaledValue and not FenceValue is redundant.